firmware/application/hw/max2837.cpp

```diff
@@ -233,10 +233,28 @@ void MAX2837::set_vga_gain(const int_fast8_t db) {
     flush();
 }
 
-void MAX2837::set_lpf_rf_bandwidth(const uint32_t bandwidth_minimum) {
+void MAX2837::set_lpf_rf_bandwidth_rx(const uint32_t bandwidth_minimum) {
+    _map.r.lpf_1.ModeCtrl = 0b01; /* Adress reg 2, D3-D2, Set mode lowpass filter block to Rx LPF . Active when Address 6 D<9> = 1 */
```

Comment:
nit: typo

Reply:
Not sure what you mean about that point. 
@kallanreed     Can you specify where is the typo ?   In which line ?  In code or in comments ???
thanks

Reply:
Ahh , I guess you mean “address” . Clear I will amend it